src/redux/rockets/rockets.js

```diff
@@ -0,0 +1,60 @@
+import axios from 'axios';
+
+const DISPLAY_ROCKET = './src/redux/rockets/rockets/DISPLAY_ROCKET';
+const baseUrl = 'https://api.spacexdata.com/v3/rockets';
+const RESERVE_ROCKET = './src/redux/rockets/rockets/RESERVE_ROCKET';
+const REMOVE_RESERVATION = './src/redux/rockets/rockets/REMOVE_RESERVATION';
+
+const initialState = [];
+
+export const displayRocket = () => async (dispatch) => {
+  const response = await axios.get(baseUrl);
+  dispatch({
+    type: DISPLAY_ROCKET,
+    payload: response.data,
+  });
+};
```

Comment:
- [ ] Just as you have done with the missions data when your rockets data is fetched you are expected to filter your data and avoid saving all the data in the state.

  According to the guidelines, you are required to only save `id`, `rocket_name`, `description`, and `flickr_images`. Kindly do that.

  Missions state (correct) | Rocket state (incorrect)
  :---:|:---:
  ![pere-missions](https://user-images.githubusercontent.com/54848363/177001776-ed9bee3c-26c3-4d9b-90ba-8836e44c08f4.jpg) | ![pere-filter](https://user-images.githubusercontent.com/54848363/177001716-3e86babd-7080-4718-8bdb-b257f4985162.jpg)

Reply:
@DammyShittu Please we don't understand what you mean here. Can you kindly throw more light

Reply:
- [ ] What the previous reviewer meant for you to do here is to only save the data you will be using use inside the Redux store. Right now, in the rockets reducer you are storing the entire data you are getting from the API. However, the data you are using is only `id`, `rocket_name`, `description` and `flickr_images` so it makes sense to only store this data from the API inside the store. Remember that the larger the Array, the more time your app will take to find the required data and the slower your application will be. Kindly make these changes, and if you still need further guidance I'd be more than happy to help 🌦️

Reply:
@zhadier, can we connect with you on slack please.

Reply:
Of course, my slack handle is Zeeshan Haider.